source/retryable-reads/tests/gridfs-download-serverErrors.yml

```diff
@@ -0,0 +1,152 @@
+database_name: &database_name "retryable-reads-tests"
+collection_name: &collection_name "files"
+
+data:
+    files:
+        -
+            _id: { "$oid" : "000000000000000000000003" }
+            length: 2
+            chunkSize: 4
+            uploadDate: { "$date" : "1970-01-01T00:00:00.000Z" }
+            md5: "c700ed4fdb1d27055aa3faa2c2432283"
+            filename: "length-2"
+            contentType: "application/octet-stream"
+            aliases: []
+            metadata: {}
+    chunks:
+        - { _id : { "$oid" : "000000000000000000000003" }, files_id : { "$oid" : "000000000000000000000004" }, n : 0, data : { $hex : "11223344" } }
+
+
+minServerVersion: '4.1.6' # 4.2
+
+tests:
+    -
+        description: "Download succeeds after InterruptedAtShutdown"
+        clientOptions:
+            retryReads: true
+        failPoint: &failCommand_failPoint
+            configureFailPoint: failCommand
+            mode: { times: 1 }
+            data: { failCommands: [find], errorCode: 11600 }
+        operation: &retryable_operation
+            operation: download
+            arguments:
+                id: { "$oid" : "000000000000000000000003" }
```

Comment:
Is this intended to assert that we read the file's contents properly, or are we happy with an error not having occurred? I'm OK with either, just curious what the intention was here since `result` is omitted.